packages/common/src/projects/HubProject.ts

```diff
@@ -133,6 +137,24 @@ export class HubProject
     return getEntityEditorSchemas(i18nScope, type, options);
   }
 
+  /**
+   * Convert the project entity into a card view model that can
+   * be consumed by the suite of hub gallery components
+   *
+   * @param project project entity
+   * @param context auth & portal information
+   * @param target card link contextual target
+   * @param locale internationalization locale
+   */
+  static convertToCardViewModel(
+    project: IHubProject,
+    context: IArcGISContext,
+    target: "ago" | "view" | "workspace" = "ago",
+    locale: string = "en-US"
```

Comment:
seeing locale drilled deep into this code makes me 🤨. Not sure how we avoid this, but it just kinda smells like we're entangling some UI layer concerns deep in here. Like maybe the `additionalInfo` interface has a `transform` prop, that tells the UI layer to convert the value to LocaleTime vs LocaleDate vs RelativeDate etc. 
Not something to change now, but something we should consider

Reply:
I think we should see where else we need this sort of "transform" logic - it smells kinda like some stuff we need for metrics / stats card as well, so maybe we can hoist out a pattern, and solve it consistently across the product. i'll create a quick issue